js/main.js

```diff
@@ -0,0 +1,219 @@
+let weatherInfo = {
+    loc: null,
+    K: null,
+    F: null,
+    C: null,
+    con: null,
+    icon: null, 
+    mCon: null,
+};
+
+function createElement(name, aType, cName, eId, bType, parent, text, varName) {
```

Comment:
In the future this could be an object `{name, aType, cName, eId, bType, parent, text, varName}` so these can all be named parameters and order wouldn't matter. Just something to look forward to in the future.